src/Servers/Kestrel/Core/src/Internal/Http3/Http3Stream.cs

```diff
@@ -821,7 +826,25 @@ private static Task ProcessUnknownFrameAsync()
             await OnEndStreamReceived();
         }
 
-        if (!_isMethodConnect && (_parsedPseudoHeaderFields & _mandatoryRequestPseudoHeaderFields) != _mandatoryRequestPseudoHeaderFields)
+        // https://www.rfc-editor.org/rfc/rfc8441#section-4
+        if (HttpRequestHeaders.HeaderProtocol.Count > 0)
+        {
+            if (!_isMethodConnect)
+            {
+                throw new Http3StreamErrorException(CoreStrings.@Http3MethodMustBeConnectWhenUsingProtocolPseudoHeader, Http3ErrorCode.ProtocolError);
```

Comment:
What's with the @?

Reply:
Oh, that is a remanent from an experiment. In `CoreStrings` they are defined with a @. So, I thought they were necessary.  I then removed it but guess I left this one.